ROVU/Core/src/controller/DataObject.java

```diff
@@ -1,4 +1,23 @@
 package controller;
```

Comment:
I think this should be a part of the model, what do you guys think? @Anthony1911 @nicket97

Reply:
Not really, it can either be part of Robot or Controller, as its used only in those two areas.

Reply:
Then I'd say a part of the Robot

Reply:
Agree